__tests__/product/product.mutation.test.js

```diff
@@ -12,7 +12,8 @@ const {
 
 require('dotenv').config();
 
-let productId, categoryId, subcategoryId, modelId, sameNameProductId;
+let productId; let categoryId; let subcategoryId; let modelId; let
+    sameNameProductId;
```

Comment:
Why did you change it? Please change it back

Reply:
it is formatted by eslint